src/lightning_app/cli/cmd_pl_init.py

```diff
@@ -121,7 +121,7 @@ def project_file_from_template(template_dir: Path, destination_dir: Path, templa
 def print_pretty_report(
     directory: pathlib.Path,
     ignore_patterns: Optional[List[str]] = None,
-    help_texts: Optional[Dict[str, str]] = None,
+    help_texts: Dict[str, str] = {},
```

Comment:
Let's revert this, and in the body do

`help_texts = {} if help_texts is None else help_texts`

Reply:
reverted